### ISSUE TITLE
Liquigraph.xsd cannot be parsed anymore

### ISSUE THREAD

**Issue by lorenz-nlp** [liquigraph/liquigraph#357]
I use Liquigrah since some time for performing Neo4j database migrations, and I love your tool!

Since today, when running migrations in my development, but also production environment, following exception is being given:

```
java.lang.IllegalArgumentException: Exception while reading changelog: 
	The element type "hr" must be terminated by the matching end-tag "</hr>"..

	at org.liquigraph.core.io.xml.XmlSchemaValidator.validate(XmlSchemaValidator.java:43)
	at org.liquigraph.core.io.xml.XmlSchemaValidator.validateSchema(XmlSchemaValidator.java:35)
	at org.liquigraph.core.io.xml.ChangelogParser.parseChangelog(ChangelogParser.java:65)
	at org.liquigraph.core.io.xml.ChangelogParser.parse(ChangelogParser.java:59)
	at org.liquigraph.core.api.MigrationRunner.parseChangesets(MigrationRunner.java:87)
	at org.liquigraph.core.api.MigrationRunner.runMigrations(MigrationRunner.java:68)
	at org.liquigraph.core.api.Liquigraph.runMigrations(Liquigraph.java:63)
Caused by: org.xml.sax.SAXParseException; systemId: http://www.liquigraph.org/schema/1.0/liquigraph.xsd; lineNumber: 6; columnNumber: 3; The element type "hr" must be terminated by the matching end-tag "</hr>".
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at org.liquigraph.core.io.xml.ExplicitSchemaValidator.parse(ExplicitSchemaValidator.java:74)
	at org.liquigraph.core.io.xml.ExplicitSchemaValidator.validate(ExplicitSchemaValidator.java:52)
	at org.liquigraph.core.io.xml.XmlSchemaValidator.parse(XmlSchemaValidator.java:52)
	at org.liquigraph.core.io.xml.XmlSchemaValidator.validate(XmlSchemaValidator.java:40)
	... 50 more
```

I used Liquigraph version 3.1.0. The migrations are being performed via the Java API. As nothing has changed within my code or migration changesets, it seems like the liquigraph.xsd is not valid any more?

**Comment by alexlitvinenko** [comment]
Hi, faced the same issue.
Upgrade to 3.1.1

reason: https://github.com/liquigraph/liquigraph/commit/99ecdac2e103cd57309fb79711173d51e9f0b336

**Comment by lorenz-nlp** [comment]
Thanks for your quick response. Updating to 3.1.1 indeed helps. However, already packaged applications with version 3.1.0 break and cannot be executed anymore.

**Comment by alexlitvinenko** [comment]
@lorenz-nlp, Have you faced this issue?
```
Failed to read schema document 'https://www.liquigraph.org/schema/1.0/liquigraph.xsd', because 1) could not find the document; 2) the document could not be read; 3) the root element of the document is not <xsd:schema>.
```

**Comment by lorenz-nlp** [comment]
No, the xsd-file is available, but it couldn't be parsed.

**Comment by fbiville** [comment]
Ok, I know why this happens. I enforced HTTPS on the documentation site yesterday which is why it breaks.

**Comment by fbiville** [comment]
I just unchecked the setting. This should fix the issue for 3.1.0 users.
